metis-normalization/src/main/java/eu/europeana/normalization/dates/extraction/extractors/NumericPartsDateExtractor.java

```diff
@@ -39,37 +39,40 @@ public class NumericPartsDateExtractor extends AbstractDateExtractor {
   private static final String UNKNOWN_CHARACTERS_REGEX = "[XU?-]";
 
   @Override
-  public DateNormalizationResult extract(String inputValue, DateQualification requestedDateQualification,
+  public DateNormalizationResult extract(String inputValue,
```

Comment:
also here my suggestion to update parameter name `flexibleDateBuild` to `allowDayMonthSwap`

Reply:
Done